### DISCUSSION TITLE
Missing Notification Configuration

### DISCUSSION THREAD

**Discussion by xeruf** [espanso/espanso#1272] [Feature requests and Ideas]
**Describe the Issue**
Unlike most other applications, I cannot configure espanso notifications in the KDE Plasma System Settings:
![image](https://user-images.githubusercontent.com/13354331/169784988-80c104d5-8688-42de-a5a6-cbc52208564e.png)
I dunno if it needs anything special for that, though.

I stumbled upon this as I wanted to disable or at least shorten the "Updated Keyboard Layout" notifications, since I update my layout via a script regularly, but the configuration options are very sparse: https://espanso.org/docs/configuration/options/#hiding-the-notifications

I can only disable all notifications, but I want to preserve the ones about enabling espanso, since I sometimes trigger it on/off accidentally.

**Comment by federico-terzi** [comment]
Hey @xeruf,

Thank you for the feedback! I'm not familiar with those settings, but we'll definitely investigate
In the meantime, I've opened another issue: https://github.com/federico-terzi/espanso/issues/1116

Cheers :)
